src/api/app/views/webui2/webui/repositories/index.html.haml

```diff
@@ -41,11 +41,11 @@
             = link_to('#', data: { toggle: 'modal', target: '#add-dod-repository-modal' }, class: 'nav-link') do
               %i.fas.fa-plus-circle.text-success
```

Comment:
I just realized, this should be `text-primary`, like all the other `Add [..]` links

Reply:
That's already in master. So let's not block this PR because of it.

I created a PR for it: #6497

Reply:
No worries :+1: